tests/support_server/setup.pm

```diff
@@ -283,6 +280,100 @@ sub setup_xdmcp_server {
     $xdmcp_server_set = 1;
 }
 
+sub setup_iscsi_server {
+    return if $iscsi_server_set;
+
+    # If no LUN number is specified we must die!
+    my $num_luns = get_var("NUMLUNS");
+    die "no LUN number defined, NUMLUNS variable missing?" unless $num_luns;
```

Comment:
you don't need this in two lines, we have a function `get_required_var` which does the same
see: http://open.qa/api/testapi/#_get_required_var